grafana-promtail/hooks/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+exec 2>&1
+
+exec promtail --config.file {{pkg.svc_config_path}}/config.yaml
```

Comment:
If the run hook is single line, should we put it into plan.sh `pkg_svc_run="..."` ?

Reply:
Refences: HAProxy, Nginx.

Reply:
i'm for it as long as we can specify command line arguments

Reply:
https://github.com/habitat-sh/core-plans/blob/master/nginx/plan.sh#L27 <-- no config arguments

https://github.com/habitat-sh/core-plans/blob/master/haproxy/plan.sh#L10 <-- has config arguments but it is not clear where the config file it is referencing

Reply:
@bdangit 

```
pkg_svc_run="haproxy -f ${pkg_svc_config_path}/haproxy.conf"
```

This is what I use in configuration plans.

Reply:
ah! thats perfect!